packages/react-packages/link/src/Link.styled.ts

```diff
@@ -1,40 +1,69 @@
 import styled from '@emotion/styled';
 
-import { LinkSize, LinkVariant } from '.';
+import { LinkSize, LinkColor, LinkVariant } from '.';
 
 interface LinkStyledProps {
   disabled?: boolean;
   size?: LinkSize;
   variant?: LinkVariant;
+  color?: LinkColor;
 }
 
 export const LinkStyled = styled.a<LinkStyledProps>`
-  ${({ theme, disabled, size, variant = 'primary' }) => `
-    color: ${theme.palette[variant].default};
-    text-decoration: none;
+  ${({ theme, disabled, size, variant = 'standalone', color = 'accent' }) => {
+    const computedColor = variant === 'inline' ? 'accent' : color;
```

Comment:
NO blocker.
```suggestion
    const linkColor = variant === 'inline' ? 'accent' : color;
```
linkColor is more friendlier 😄